openshift/datagrid-service/xsite/README.md

```diff
@@ -138,6 +136,13 @@ $ oc new-app datagrid-service \
   `${extAddr-SiteA}[${extPort-SiteB}],${extAddr-SiteB}[${extPort-SiteA}]`
     For example: `192.0.2.0[32660],192.0.2.1[32660]`
 
+  **NOTE:** The launch script does not generate keystores when you create Data Grid with custom configuration. To enable authentication, you must set up keystores for `datagrid-service` manually and include the following parameters:
```

Comment:
Btw, this also applies to the `user-config` tutorial. Shall we add same info there too?